lib/danger_plugin.rb

```diff
@@ -39,33 +39,67 @@ def lint_files
         return
       end
 
-      swiftlint_command = ""
-      swiftlint_command += "cd #{directory} && " if directory
-      swiftlint_command += "swiftlint lint --quiet --reporter json"
-      swiftlint_command += " --config #{config_file}" if config_file
-
-      require 'json'
-      result_json = JSON.parse(`(#{swiftlint_command})`).flatten
-
-      # Convert to swiftlint results
-      warnings = result_json.select do |results| 
-        results['severity'] == 'Warning'
-      end
-      errors = result_json.select do |results| 
-        results['severity'] == 'Error' 
-      end
-
-      message = ''
-
-      # We got some error reports back from swiftlint
-      if warnings.count > 0 || errors.count > 0
-        message = "### SwiftLint found issues\n\n"
+      require 'tempfile'
```

Comment:
I wouldn't usually put `require` inline, but that's what I've seen for the `json` required down in this method.